Mrinali/lab02/part2/hellolib/include/hello_world.hpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#pragma once
```

Comment:
typically the `#pragma once` is the first line with every other include below the directive:

https://learn.microsoft.com/en-us/cpp/preprocessor/once?view=msvc-170